src/DotNetWorker.Core/Http/HttpCookie.cs

```diff
@@ -10,6 +11,25 @@ namespace Microsoft.Azure.Functions.Worker.Http
     /// </summary>
     public sealed class HttpCookie : IHttpCookie
     {
+        private KeyValuePair<string, string> c;
```

Comment:
Changes in this file were added/populated automatically - I didn't touch this file.